modules/mono/glue/GodotSharp/GodotPlugins/GodotPlugins.csproj

```diff
@@ -8,6 +8,7 @@
 
         <!-- To generate the .runtimeconfig.json file-->
         <EnableDynamicLoading>true</EnableDynamicLoading>
+        <RollForward>LatestMajor</RollForward>
```

Comment:
Have you tried using `Major` instead? AFAIK that should prefer the .NET 6 SDK when both are installed.

Reply:
Yes, but that means you can't use projects targeting .net7 in the editor if both .net6 and .net7 are installed on a machine, but after uninstalling .net6 it will then work, which is IMO really unintuitive.

Reply:
:thinking: Oh yeah, I agree it's intuitive.

Reply:
We talked about this the other day. The problem with `Major` is that (assuming both net6 and net7 are installed) if `GodotPlugins` targets net6, then that's what the Godot editor will load. If that happens, there will be problems if a game project targets net7, because only one runtime can be loaded.
As such, we decided `LatestMajor` was the way to go. The only drawback here is the risk of a future .NET version breaking backwards compatibility. I think this is a risk we will have to live with, but it would be great if we can have a workaround for users in case that ever happens. My idea was to add a project/editor setting to use `Major` if desired.

Reply:
The question really is how long term .NET 6 support is supposed to be. If the plan is to fully switch to .NET 7 as soon as all the kinks are worked out, I would just not bother with being able to use 6 over 7, when both are installed. If on the other hand .NET 6 is going to remain supported for the full 4.0 release, I would load the framework based on the runtime config of the user project and only fall back to the runtime config of GodotPlugins, if the project was not built yet / the desired target framework is not installed.